### PR TITLE
Add community profile

### LINKED ISSUE #16: Add Community Profile
**Is your request related to a problem? Please describe.**
The community profile is missing. And it would be nice to add the community standards.

**Describe the solution you'd like**
Add the following files to follow the recommended community standards:
- [x] Description
- [x] Code of Conduct
- [x] Contributing
- [x] [License](https://choosealicense.com/)

### PR DESCRIPTION
**Pull request by flaxel**
**What has been done?**

- add Description, Code of Conduct, Contributing, License

**Why do you changed it?**

- missing components for the community profile

**Instructions for testing**

- check if all components are there

**Checks**
- [ ] I reviewed my own code
- [ ] I have documented my code
- [ ] I have tested my code
- [x] I have linked the issue to the pull request

### CODE REVIEW

**Comment by flaxel** [comment]
@fgr Can you take a look if you would do something else?

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at help.flaxel@gmail.com. All
```
Comment (flaxel) [inline .github/CODE_OF_CONDUCT.md]: Could we use another mail?

**Review by fgr — DISMISSED** [review]
I'm okay with the proposed changed. We might have to switch to a different license for the VLINGO folks to accept our project.

**Comment by flaxel** [comment]
Is it better to switch directly to the Mozilla license?

**Review by Florian122 — APPROVED** [review]

**Pull request merged**